mtp_api/apps/core/management/commands/delete_zendesk_tickets.py

```diff
@@ -0,0 +1,42 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management import BaseCommand
+from django.utils import timezone
+import requests
+
+SEARCH_URL = settings.ZENDESK_BASE_URL + '/api/v2/search.json'
+DELETE_URL = settings.ZENDESK_BASE_URL + '/api/v2/tickets/destroy_many.json'
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        auth = (
+            '%s/token' % settings.ZENDESK_API_USERNAME,
+            settings.ZENDESK_API_TOKEN
+        )
+
+        query = 'type:ticket status:closed group_id:{group_id} updated<{two_weeks}'.format(
+            group_id=settings.ZENDESK_GROUP_ID,
+            two_weeks=(timezone.now() - timedelta(weeks=2)).strftime('%Y-%m-%d')
+        )
+        url = SEARCH_URL + '?page=1&query=%s' % query
+
+        while True:
+            run_again = False
+            response = requests.get(url, auth=auth, timeout=15)
```

Comment:
why not pass `page` and `query` as a dict, that way escaping : and < etc is done automatically